odl/space/entry_points.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2014-2017 The ODL contributors
+# Copyright 2014-2018 The ODL contributors
```

Comment:
Actually I tried to update only if some other part of the file was changed as well. This one slipped